src/plugins/intel_gna/legacy/src/convert_function_to_cnn_network.cpp

```diff
@@ -353,11 +353,13 @@ CNNLayer::Ptr createSubGraphLayer(const std::shared_ptr<ngraph::Node>& layer) {
  */
 class CNNLayerCreator : public ::ngraph::AttributeVisitor {
 public:
+    explicit CNNLayerCreator();
+
+    CNNLayerPtr create(const std::shared_ptr<::ngraph::Node>& origin);
```

Comment:
origin -> current_node?

Reply:
I don't see previous or next here, so current looks to be redundant info here. having 'node' in field name while you do have access to type in function definition (I mean it's not "auto origin") also seemed to be a bit redundant to me. Additionally when you look at call stack, You'll see following:

        CNNLayerPtr cnnLayer = createCNNLayer(layer);

that's why I finally decided that 'origin' is enough.